WordPressMocks.podspec

```diff
@@ -1,11 +1,19 @@
 Pod::Spec.new do |s|
-  s.name           = 'WordPressMocks'
-  s.version        = '0.0.9'
-  s.platform       = :ios
-  s.summary        = 'Network mocking for testing the WordPress mobile apps.'
-  s.homepage       = 'https://github.com/wordpress-mobile/WordPressMocks'
-  s.license        = { type: 'GPLv2', file: 'LICENSE.md' }
-  s.author         = { 'James Treanor' => 'jtreanor3@gmail.com' }
-  s.source         = { git: "https://github.com/wordpress-mobile/WordPressMocks.git", :tag => s.version.to_s }
+  s.name          = 'WordPressMocks'
+  s.version       = '0.0.9'
+
+  s.summary       = 'Network mocking for testing the WordPress mobile apps.'
+  s.description   = <<-DESC
+                    This framework contains a pack of icons – mainly ones used in the WordPress apps –
+                    that can be reused and scaled at any resolution.
+                  DESC
+
+  s.homepage      = 'https://github.com/wordpress-mobile/WordPressMocks'
+  s.license       = { type: 'GPLv2', file: 'LICENSE.md' }
+  s.author        = { "The WordPress Mobile Team" => "mobile@automattic.com" }
+
+  s.platform      = :ios
+
+  s.source        = { git: "https://github.com/wordpress-mobile/WordPressMocks.git", :tag => s.version.to_s }
   s.preserve_paths = 'WordPressMocks/src', 'scripts'
```

Comment:
Note: Unlike the others, this pod doesn't have an `s.swift_version` entry right after `s.platform`, because it doesn't have any Swift source file.
In fact, it doesn't have any source file at all as this is only composed of WireMock `.json` files – which is why, in contrast, we have `s.preserve_paths` to keep those non-source-files around instead.